FeatureDetect/src/source/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.22.3)
+#project(source)
+#set(MY_SOURCE_LIB featurelib.cpp)
+#add_library(source STATIC ${MY_SOURCE_LIB})
+#include_directories(source)
+set(SOURCE_FLS featurelib.cpp)
```

Comment:
Don't shorten words like this where unnecessary -- I'm speaking about `FLS`

Reply:
done